pkg/util/feature/feature_gate.go

```diff
@@ -31,3 +33,13 @@ var (
 	// Top-level commands/options setup that needs to modify this feature gate should use DefaultMutableFeatureGate.
 	DefaultFeatureGate featuregate.FeatureGate = DefaultMutableFeatureGate
 )
+
+func EnableFeatureGate(featureGate featuregate.Feature) error {
+	DefaultMutableFeatureGate.Set(fmt.Sprintf("%s=%t", featureGate, true))
+	return nil
+}
+
+func DisableFeatureGate(featureGate featuregate.Feature) error {
```

Comment:
Please remove both of these functions - they aren't needed

Reply:
ok, now using SetFeatureGateDuringTest instead